README.md

```diff
@@ -2132,6 +2132,58 @@ export async function loader({ request }: LoaderArgs) {
 }
 ```
 
+### Detect if Application is Stale
+
+> **Note**
+> This depends on `react` and `@remix-run/react`.
+
+The `useAppIsStale` hook lets you detect if the application client is stale compared to the server version, this means the user is seeing an old version of the application and should reload the page.
+
+```tsx
+import { useAppIsStale } from "remix-utils/use-app-is-stale";
```

Comment:
Not sure, but you may also need to update `exports` field from `package.json` to make this available